packages/shared/src/types.ts

```diff
@@ -13,7 +13,7 @@ export interface WinningLine {
 
 export type GameStatus = {
   board: Board;
-  winner: keyof typeof Winner | null;
+  winner: typeof Winner.o | typeof Winner.x | typeof Winner.tie | null;
```

Comment:
Consequence of changing the value in the enum, but felt less coupled. Not super confident in this change vs the CSS transform.